src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_CallerArgumentExpression.vb

```diff
@@ -254,13 +254,8 @@ Module Program
 End Module
 "
 
-            Dim compilation = CreateCompilation(source, targetFramework:=TargetFramework.NetCoreApp, references:={Net451.MicrosoftVisualBasic}, options:=TestOptions.ReleaseExe, parseOptions:=TestOptions.Regular16_9)
-            compilation.AssertTheseDiagnostics(
-<expected>
-BC36716: Visual Basic 16.9 does not support caller argument expression.
-        Log(123)
-            ~~~
-</expected>)
+            Dim compilation = CreateCompilation(source, targetFramework:=TargetFramework.NetCoreApp, references:={Net451.MicrosoftVisualBasic}, options:=TestOptions.ReleaseExe, parseOptions:=TestOptions.Regular16_9.WithFeature("CallerArgumentExpression"))
```

Comment:
Rather than modifying this test to succeed, let's leave the feature flag off and verify the behavior when it's not present.